src/routes/bounce_rules_detailed.jsx

```diff
@@ -26,8 +36,126 @@ export default class BounceRuleDetailedPage extends React.Component {
     }
   }
 
+  onChangeRule(event, field) {
+    const { currentRule } = this.state;
+    const newRule = Object.assign({}, currentRule);
+    newRule[field] = event.target.value;
+
+    this.setState({
+      currentRule: newRule,
+    });
+  }
+
+  handleModalClose(modal) {
+    switch (modal) {
+      case "changeModal": {
+        this.setState({
+          isChangeModalOpen: false,
+        });
+        break;
+      }
+      case "cancelModal": {
+        this.setState({
+          isCancelConfirmOpen: false,
+        });
+        break;
+      }
+      case "saveModal": {
+        this.setState({
+          isConfirmOpen: false,
+        });
+        break;
+      }
+      default: {
+        throw Error("Modal Not Found");
```

Comment:
We don't need to throw an Error here but i'd say for failed asynchronous requests

Reply:
ES Lint would give me an error saying I needed a default, what should I have here instead of an Error :o ?